README.md

```diff
@@ -25,7 +25,8 @@
   be annotated. The annotations should be removed after the translations are
   completed. There are three translation formats: `"TRANSLATION MISSING", TRANSLATION\ MISSING, TRANSLATION MISSING`. `"TRANSLATION MISSING"`
```

Comment:
Break these three format into separate snippets:

`"TRANSLATION MISSING"`, `TRANSLATION\ MISSING`, `TRANSLATION MISSING`

Wrap them to limit the line width before 80-column margin.